### ISSUE TITLE
as.ctd documentation needs updating (at least on website)

### ISSUE THREAD

**Issue by richardsc** [dankelley/oce#1788]
Was just looking at the `as.ctd()` documentation on the pkgdown website at https://dankelley.github.io/oce/reference/as.ctd.html and I see as below:
![image](https://user-images.githubusercontent.com/233584/109509943-d3eaa800-7a77-11eb-9dac-2017b07594f8.png)

The question is, "***CAN*** the `salinity` argument be an `rsk` object????" 😄

**Comment by dankelley** [comment]
Not sure I'm following.  Is it about the wording or the action?  The action should do it, according to the following first lines of `as.ctd()`:
```R
    if (!missing(salinity) && inherits(salinity, "rsk")) {
        oceDebug(debug, "as.ctd(...) {\n", sep="", unindent=1)
        res <- rsk2ctd(salinity,
                       pressureAtmospheric=pressureAtmospheric,
                       longitude=longitude,
                       latitude=latitude,
                       ship=ship,
                       station=station,
                       cruise=cruise,
                       deploymentType=deploymentType,
                       debug=debug-1)
        oceDebug(debug, "} # as.ctd()\n", sep="", unindent=1)
        return(res)
    }
```

**Comment by dankelley** [comment]
Oh, I see. It's about the repeated lines in the docs.  I've updated "develop" commit 7341de94f9d1136e883916afb39c62330c0ade8d.  I also merged "develop" into "master" and (I think) that means that the website will get updated within about half an hour.

**Comment by dankelley** [comment]
Hm, the pkgdown action failed (https://github.com/dankelley/oce/runs/2006096171?check_suite_focus=true) so I imagine the webpage did not update correctly.

I don't know what's wrong with the action and I don't have time to try to debug it.  (These actions are wonderful but when they fail, it's hard to figure out why.  And sometimes, it's a problem unrelated to the package, and so the build starts working a week later ... we had that with argoFloats a few days ago.)

**Comment by richardsc** [comment]
Yes, sorry -- I should have been clearer but I was trying to be funny 😆 .

Not sure what's going with the action (TBH I didn't know we had a webpage action for this now!), but I'll close because my issue wasn't with the webpage so much as with the actual documentation (though I have to say I LOVE having the pkgdown website as a reference -- I often go to it before even firing up R and typing e.g. `?as.ctd`)

**Comment by dankelley** [comment]
I'm kind of the reverse.  I often do e.g.
```
R -e "?oce::as.ctd"
```
at the CLI.  Since R loads in under a second, this is a quick way to get text-based docs.  Since I often read the docs in a vim window anyway, I'm happy with this presentation.

PS. I just pushed another version.  There was *still* a remaining `rsk` reference.  Sheesh.  If I used emacs instead of vim, I bet some fancy lisp code would have noticed the similarity of sentences and auto-corrected for me, or launched "doctor" for me.  Humour.  Smiley cat face. 😸
